lib/definitions/server/server-accounts-service.d.ts

```diff
@@ -89,3 +92,14 @@ interface IUsageInfo extends IUsageInfoBase {
 	 */
 	softUsageLimit: number;
 }
+
+interface IFeatureInfo extends IFeatureInfoBase {
+	/**
+	 * The type of the feature (e.g. CloudBuilds).
+	 */
+	type: string;
+	/**
+	 * If this property is set to true the allowed usage is unlimited.
+	 */
+	enabled: boolean;
```

Comment:
this name is really misleading based on the JSDocs above. Is it possible to name it `hasUnlimitedUsage` for example? Ping @TsvetanMilanov

Reply:
The name of the property is correct. The JSDocs description should be changed.